testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -1,76 +1,85 @@
+#!/usr/bin/env python
+
 import numpy as np
 import xarray as xr
 import matplotlib
 matplotlib.use('Agg')
 import matplotlib.pyplot as plt
-import os
 
 dpi = 200
 
+
 def rmse(resTag):
-# resTag is the resolution to compute RMSE
-# sliceTag is the time slice, assumes 1 hour output and same for all cases
-    fid = open('../../../{}/default/init_step/namelist.ocean'.format(resTag),'r')
+    """
+    resTag is the resolution to compute RMSE
+    """
+
+    fid = open('../../../{}/default/init_step/namelist.ocean'.format(resTag),
+               'r')
     temp = fid.readlines()
 
     fid.close()
     for i, line in enumerate(temp):
-        if "config_cosine_bell_lat_center" in line: 
+        if "config_cosine_bell_lat_center" in line:
             ii = line.find('=')+1
             latCent = float(line[ii:])
-        if "config_cosine_bell_lon_center" in line: 
+        if "config_cosine_bell_lon_center" in line:
             ii = line.find('=')+1
             lonCent = float(line[ii:])
-        if "config_cosine_bell_radius" in line: 
+        if "config_cosine_bell_radius" in line:
             ii = line.find('=')+1
             radius = float(line[ii:])
-        if "config_cosine_bell_psi0" in line: 
+        if "config_cosine_bell_psi0" in line:
             ii = line.find('=')+1
             psi0 = float(line[ii:])
-        if "config_cosine_bell_vel_pd" in line: 
+        if "config_cosine_bell_vel_pd" in line:
             ii = line.find('=')+1
             pd = float(line[ii:])
 
-    init = xr.open_dataset('../../../{}/default/init_step/initial_state.nc'.format(resTag))
-    #find time since the beginning of run
-    ds = xr.open_dataset('../../../{}/default/forward/output/output.0001-01-01_00.00.00.nc'.format(resTag))
+    init = xr.open_dataset('../../../{}/default/init_step/initial_state.nc'
+                           ''.format(resTag))
+    # find time since the beginning of run
+    ds = xr.open_dataset('../../../{}/default/forward/output/'
+                         'output.0001-01-01_00.00.00.nc'.format(resTag))
     for j in range(len(ds.xtime)):
-      tt=str(ds.xtime[j].values)
-      tt.rfind('_')
-      DY = float(tt[10:12])-1
-      if DY == pd:
-        sliceTime = j
-        break
+        tt = str(ds.xtime[j].values)
+        tt.rfind('_')
+        DY = float(tt[10:12])-1
+        if DY == pd:
+            sliceTime = j
+            break
```

Comment:
Put spaces around equals signs except for keyword arguments to functions (in which case, never put spaces).

Always indent by a multiple of 4 spaces.